pkg/sql/schemachanger/scbuild/internal/scbuildstmt/alter_table_add_constraint.go

```diff
@@ -38,3 +75,240 @@ func alterTableAddConstraint(
 		StorageParams: d.StorageParams,
 	})
 }
+
+// alterTableAddConstraintCheck contains logic for building ALTER TABLE ... ADD CONSTRAINT ... CHECK.
+// It assumes `t` is such a command.
+func alterTableAddConstraintCheck(
```

Comment:
`alterTableAddCheckConstraint`

Reply:
I changed it to just `alterTableAddCheck`, in accordance to `alterTableAddPrimaryKey`.